packages/runner-docker/Dockerfile

```diff
@@ -26,6 +24,10 @@ RUN git clone https://github.com/brendangregg/FlameGraph.git
 VOLUME ["/home/node/repo"]
 VOLUME ["/home/node/results"]
 EXPOSE 3000 9229
+
+# Create results directory with proper ownership
+RUN mkdir -p /home/node/results && chown -R node:node /home/node/results
```

Comment:
I am pretty sure that docker creates this for the mounts. There is no other case where this would run that is not using the local mounts, so can you help me understand this change?